initScripts/x86_64/WindowsServer_2016/boot.ps1

```diff
@@ -56,7 +56,8 @@ $REQPROC_CONTAINER_NAME = "$REQPROC_CONTAINER_NAME_PATTERN-$BASE_UUID"
 $REQKICK_SERVICE_NAME_PATTERN = "shippable-reqKick@"
 
 # TODO: update container directories while mounting
-$DEFAULT_TASK_CONTAINER_MOUNTS = "-v ${BUILD_DIR}:${CONTAINER_BUILD_DIR} -v ${REQEXEC_DIR}:${CONTAINER_REQEXEC_DIR}"
+$DEFAULT_TASK_CONTAINER_MOUNTS = "-v ${BUILD_DIR}:${CONTAINER_BUILD_DIR} -v ${REQEXEC_DIR}:${CONTAINER_REQEXEC_DIR} " + `
+"-v ${NODE_SCRIPTS_LOCATION}:'C:\Users\Administrator\Shippable\node'"
```

Comment:
could we save the path in an env like `CONTAINER_NODE_DIR` and use it here, instead of hardcoding? (just to maintain uniformity)